CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contribute
+
+## Setting up
+
+1. Clone this git repository on your local machine inside a `wp-content/plugins/` directory
```

Comment:
Perhaps:  "1. Clone this git repository to your development system, this is commonly inside your WordPress development site in the 'wp-content/plugins' directory."